.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 node_js:
-  - "6"
   - "8"
   - "10"
   - "12"
+  - "node"
```

Comment:
we should be explicit here

Reply:
> keep in mind that this is a breaking change and needs a major version bump

Exactly.

Reply:
> we should be explicit here

Why not fail early?

Reply:
because we'd lose coverage for 13 once 14 is released

Reply:
This is fine imo as odd release numbers are shortliving anyways.